dom/src/main/scala-2/fs2/dom/Dom.scala

```diff
@@ -112,6 +112,40 @@ object Element {
 }
 
 trait HtmlElement[F[_]] extends Element[F]
+object HtmlElement {
+
+  implicit def ops[F[_]](element: HtmlElement[F]): Ops[F] =
+    new Ops(element.asInstanceOf[dom.HTMLElement])
+
+  final class Ops[F[_]] private[HtmlElement] (private val element: dom.HTMLElement) extends AnyVal {
+    def focus(implicit F: Dom[F]): F[Unit] = F.delay(element.focus())
+
+    // facade for overload missing from org.scalajs.dom
+    def focus(options: FocusOptions)(implicit F: Dom[F]): F[Unit] =
```

Comment:
We can bump to latest scala-js-dom and use the `FocusOptions` facade instead of a case class.